services/wafv2.py

```diff
@@ -2,28 +2,83 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("wafv2", region_name="us-east-1")
+
+cloudfront_client = boto3.client("cloudfront", "us-east-1")
```

Comment:
WAFv2는 regional resource와 global(CloudFront) resource가 있어서 두 개의 client가 별도로 필요할 수 있습니다.
```suggestion
client = boto3.client("wafv2")
global_client = boto3.client("wafv2", region_name="us-east-1")
```